### PR TITLE
COR-1515-remove-VR-pages-part-II

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard18**
## Summary

* 
* the removal of:
* /rioolwater
* /sterfte
* /thuiswonende-ouderen
* /vaccinaties
* /ziekenhuis-opnames
* 

#### After
<details>
<summary>Toggle after screenshots</summary>

Screenshot 2023-03-22 at 13 14 01

Screenshot 2023-03-22 at 13 14 03

Screenshot 2023-03-22 at 13 14 06

Screenshot 2023-03-22 at 13 14 09

Screenshot 2023-03-22 at 13 13 58

</details>

### CODE REVIEW

**Review by VWSCoronaDashboard26 — APPROVED** [review]

File: packages/app/src/pages/veiligheidsregio/[code]/thuiswonende-ouderen.tsx [hunk 1]
```diff
@@ -1,273 +0,0 @@
...
-          <ChartTile
-            timeframeOptions={TimeframeOptionsList}
-            title={textVr.section_deceased.line_chart_daily_title}
-            metadata={{ source: textVr.section_positive_tested.bronnen.rivm }}
-            description={textVr.section_deceased.line_chart_daily_description}
-            onSelectTimeframe={setElderlyAtHomeConfirmedCasesOverTimeTimeframe}
-          >
-            <TimeSeriesChart
-              accessibility={{
-                key: 'elderly_at_home_deceased_over_time_chart',
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/pages/veiligheidsregio/[code]/thuiswonende-ouderen.tsx:234]: This Sanity key can also be removed.

**Review by VWSCoronaDashboard26 — DISMISSED** [review]

File: packages/cms/src/lokalize/key-mutations.csv [hunk 2]
```diff
@@ -45,3 +199,5 @@ timestamp,action,key,document_id,move_to
 2023-03-24T07:45:03.890Z,delete,pages.topical_page.shared.secties.search.title.nl,PGwiP4IwXrA0GfLCLD9Tth,__
 2023-03-24T07:45:03.890Z,delete,pages.topical_page.shared.secties.search.title.vr,7nClfXqPlyZyEU7PO2TAAw,__
 2023-03-23T08:50:33.174Z,delete,common.nav.links.veiligheidsregio,jF33EuwumlGuwav2FD3VW8,__
+2023-03-27T13:01:45.444Z,delete,common.accessibility.charts.elderly_at_home_deceased_over_time_chart.description,G1DXw0RdifOml06twMjSTZ,__
```
Comment (VWSCoronaDashboard21) [inline packages/cms/src/lokalize/key-mutations.csv:202]: Aren't the accessibility keys not needed for the Elderly at home page on NL level?

Comment (VWSCoronaDashboard18) [reply packages/cms/src/lokalize/key-mutations.csv:202]: No, they are not. Please resolve this.

**Review by VWSCoronaDashboard21 — APPROVED** [review]

**Review by VWSCoronaDashboard21 — APPROVED** [review]
LGTM!

**Review by VWSCoronaDashboard26 — APPROVED** [review]

**Pull request merged**